docker/web/celery/scheduler/run.sh

```diff
@@ -13,4 +13,6 @@ fi
 sleep 10
 
 echo "==> $(date +%H:%M:%S) ==> Running Celery beat <=="
-exec celery -C -A config.celery_app beat -S django_celery_beat.schedulers:DatabaseScheduler --loglevel $log_level
+exec celery -C -A config.celery_app beat \
```

Comment:
`without-heartbeat` and so on are not required for the scheduler?

Reply:
looks like is a flag accepted only for workers.
`safe-transaction-service-scheduler-1  | Error: No such option: --without-heartbeat`